lib/render/ParticleRenderer.cpp

```diff
@@ -42,6 +43,8 @@
 
 using namespace VAPoR;
 
+//#define DEBUG
+
```

Comment:
Is there any specific reason to add two lines of code that doesn't do anything?

Reply:
I like to have commented switches like this because I can easily turn them on/off without needing to find the macro keyword in the code.  But no biggie.  Removed.